sqlserver/README.md

```diff
@@ -28,10 +28,17 @@ _Server Properties_ -> _Security_ -> _SQL Server and Windows Authentication mode
         GRANT SELECT on sys.dm_os_performance_counters to datadog;
         GRANT VIEW SERVER STATE to datadog;
     ```
+   
+   If you want to collect file size metrics per database, ensure the `datadog` user has access to your databases.
```

Comment:
This would be the user specified in `username`, not necessarily `datadog`, right?

Reply:
Yes, this is just using the username we ask them to create in the example above. I can replace with just `username`